docs/fedora-releases-guide/index.md

```diff
@@ -253,3 +253,28 @@ you can override this behaviour by specifying
 `allowed_builders` in the [job configuration](/docs/configuration/downstream/bodhi_update).
 
 For a configuration example and retriggering, see [dist-git onboarding](./dist-git-onboarding.md#bodhi-update-job).
+
+
+# Permission Handling
+
+The downstream automation is a chain of jobs: propose downstream/pull from upstream -> koji build -> bodhi update.
```

Comment:
```suggestion
The downstream automation is a pipeline of jobs: propose_downstream/pull_from_upstream -> koji_build -> bodhi_update.
```

Technically, the first part is optional, one can start a pipeline with a direct push. Should we mention that somehow?

Reply:
What about something like this:


```suggestion
The downstream automation is a pipeline of jobs: propose_downstream/pull_from_upstream/your-own commits -> koji_build -> bodhi_update.
```

Reply:
But it could also be Packit commits pushed by a user :slightly_smiling_face: 
Actually, it should look more like this:
`propose_downstream/pull_from_upstream -> Packit PR merge/dist-git push -> koji_build -> bodhi_update.`

Reply:
Or perhaps this is more accurate:
```
propose_downstream/pull_from_upstream -> Packit PR merge -> koji_build -> bodhi_update
dist-git push -> koji_build -> bodhi_update
```